azimuth/routers/v1/utterances.py

```diff
@@ -212,23 +212,32 @@ def get_utterances(
     else:
         model_saliencies = [None] * len(ds)
 
-    available_tags = (
-        ALL_SMART_TAGS
-        if pipeline_index is not None
-        else set(ALL_SMART_TAGS).difference(ALL_PREDICTION_TAGS)
-    )
+    if pipeline_index is not None:
+        available_families = list(SMART_TAGS_FAMILY_MAPPING.keys())
+    else:
+        available_families = [
+            SmartTagFamily.extreme_length,
+            SmartTagFamily.partial_syntax,
+            SmartTagFamily.dissimilar,
+        ]
```

Comment:
Can we reuse our new constants here?
```suggestion
    available_families = SMART_TAG_FAMILY_DATASET
    if pipeline_index is not None:
        available_families += SMART_TAG_FAMILY_PIPELINE
```